### PR TITLE
DevTools 106 > console.createTask

### PR DESCRIPTION
**Pull request by sofiayem**
Documented https://developer.chrome.com/blog/devtools-modern-web-debugging/#linked-stack-traces

### CODE REVIEW

**Comment by sofiayem** [comment]
Hi @victorporof! Can you please fact-check this?

File: site/en/docs/devtools/console/api/index.md [hunk 1]
```diff
@@ -73,6 +73,24 @@ console.countReset();
 console.countReset('coffee');
 ```
 
+## console.createTask(name) {: #createtask }
+
+{% Aside 'gotchas' %}
+This method implements the Async Stack Tagging API. If you're using a framework that handles the scheduling and async execution, you don't need to use this method. In that case, it is up to the framework to implement the API.
```
Comment (victorporof) [inline site/en/docs/devtools/console/api/index.md:79]: I'd word this as "This method is known as the Async Stack Tagging API. It you're using a framework or abstraction that already started using this under the hood to keep track of scheduling and async execution, then you don't need to use this method directly."

File: site/en/docs/devtools/console/api/index.md [hunk 2]
```diff
@@ -73,6 +73,24 @@ console.countReset();
...
+{% endAside %}
+
+Returns a `Task` instance that associates the current async stack trace with the created `task`. You can later use the `task` to run its content (`f` in the following example). The `task.run(f)` executes an arbitrary payload and forwards the return value back to the caller.
```
Comment (victorporof) [inline site/en/docs/devtools/console/api/index.md:82]: I'd word this as "Returns a `Task` instance that associates the current stack trace with the created `task` object. You can later use this `task` object to run a function (`f` in the following example). The `task.run(f)` executes an arbitrary payload and forwards the return value back to the caller."

**Review by victorporof — CHANGES_REQUESTED** [review]
Looks good with some wording nits. Thanks!

File: site/en/docs/devtools/console/api/index.md (hunk 1 above)
Comment (victorporof) [reply site/en/docs/devtools/console/api/index.md:79]: I'd remove "In that case, it is up to the framework to implement the API.". Frameworks don't _implement_ this API, they use it. We (DevTools) implemented the API.

**Review by victorporof — APPROVED** [review]

**Pull request merged**